### PR TITLE
Adding Spanish Translation for the Pyladies Berlin Website

### LINKED ISSUE #87: [Content] Translation for ES
We wold like to add Spanish language for the current content.

### PR DESCRIPTION
**Pull request by mjmolina**
Hello!

**Goal of this PR:** Website translation to Spanish.

fixes #87

### CODE REVIEW

**Review by sleepypioneer — APPROVED** [review]
Muy Bien, Gracias! 🙌 🇪🇸

File: content/pages/es/about.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+:order: 1
+:date: 2024-05-21 
+:icon: icon-link2
+:summary:
+:lang: es
+:slug: about
+
+Sobre PyLadies Berlin:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PyLadies es un grupo internacional de tutoría para géneros marginados, como personas no binarias, personas trans y mujeres en la tecnología, con el objetivo de ayudar a nuestra comunidad a convertirse en participantes activos y líderes en la comunidad de código abierto de Python. Todos los niveles son bienvenidos!
```
Comment (cmaureir) [inline content/pages/es/about.rst]:
```suggestion
PyLadies es un grupo internacional de tutoría para géneros marginados, como personas no binarias, personas trans y mujeres en la tecnología, con el objetivo de ayudar a nuestra comunidad a convertirse en participantes activos y líderes en la comunidad de código abierto de Python. ¡Todos los niveles son bienvenidos!
```

File: content/pages/es/about.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Política de género:
+----------------------
+Creemos que el conocimiento es para todos. Al mismo tiempo, nuestros eventos tienen como objetivo principal empoderar a las mujeres y a las minorías de género en la comunidad tecnológica. Por ello, si no te identificas como parte de ninguna de ellas, te pedimos que seas consciente de esta situación y hagas que tu presencia sea discreta, por ejemplo, viniendo con una mujer o una persona de una minoría de género para garantizar el equilibrio de género, y que consideres que no se te escucha más ni ocupas más espacio que los demás en los debates y secciones de preguntas.
```
Comment (cmaureir) [inline content/pages/es/about.rst]:
```suggestion
Creemos que el conocimiento es para todas las personas. Al mismo tiempo, nuestros eventos tienen como objetivo principal empoderar a las mujeres y a las minorías de género en la comunidad tecnológica. Por ello, si no te identificas como parte de ninguna de ellas, te pedimos que seas consciente de esta situación y hagas que tu presencia sea discreta, por ejemplo, viniendo con una mujer o una persona de una minoría de género para garantizar el equilibrio de género, y que consideres que no se te escucha más ni ocupas más espacio que los demás en los debates y secciones de preguntas.
```

File: content/pages/es/about.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Aquí te explicamos cómo puedes participar:
+-------------------------------------------
+
+* Participa en nuestros eventos, asistiendo o como mentor `Meetup <https://www.meetup.com/de-DE/PyLadies-Berlin/>`__
+* Únete al espacio de trabajo de PyLadies `Slack <https://slackin.pyladies.com>`__ 
+* Pregúntanos!
```
Comment (cmaureir) [inline content/pages/es/about.rst]:
```suggestion
* ¡Pregúntanos!
```

File: content/pages/es/jobs.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+.. container:: float-right
+
+    .. image:: {attach}/images/jobs/poetry-workshop.JPG
+        :width: 300px
+        :alt: Taller de instalación de Poetry en PyLadies Berlín
+
+
+**¿Estás buscando un trabajo relacionado con Python?**
+
+Echa un vistazo a nuestro `slack`_ en el canal #jobs-europe canal y #jobs (este último canal tiene sobre todo Estados Unidos puestos de trabajo)
```
Comment (cmaureir) [inline content/pages/es/jobs.rst]:
```suggestion
Echa un vistazo a nuestro `slack`_ en el canal #jobs-europe canal y #jobs (este último canal tiene puestos de trabajo en Estados Unidos)
```

File: content/pages/es/jobs.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+**¿Tu empresa tiene buenos puestos en Python que quieras compartir?**
+
+  - Publica el trabajo directamente en nuestro `slack`_ en el canal #job-europe.
```
Comment (cmaureir) [inline content/pages/es/jobs.rst]:
```suggestion
  - Publica el trabajo directamente en nuestro `slack`_ en el canal #jobs-europe.
```

File: content/pages/es/meetups.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
+Meetups
+#######
+
+:order: 2
+:date: 2024-05-21
+:icon: icon-speech-bubble
+:summary: Meetups mensual de Pyladies Berlin
```
Comment (cmaureir) [inline content/pages/es/meetups.rst]:
```suggestion
:summary: Meetups mensual de PyLadies Berlin
```

File: content/pages/es/meetups.rst [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        :width: 350px
+
+Meetups
+~~~~~~~
+
+En nuestro Meetup mensual podrás asistir a talleres y charlas sobre diversos temas relacionados con Python.
+temas relacionados con Python, como el desarrollo de software o la ciencia de datos.
+
+Además, cubrimos temas básicos en nuestros eventos de configuración que harán tu
+empezar a programar con Python.
```
Comment (cmaureir) [inline content/pages/es/meetups.rst]:
```suggestion
empezar a programar con Python más fácil.
```

File: content/pages/es/meetups.rst [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+Puedes encontrar y unirte a los próximos eventos en nuestro grupo `Meetup
+<https://www.meetup.com/PyLadies-Berlin/>`__.
+
+Tenemos diferentes niveles en los diferentes meetups desde principiante hasta experto en
+temas específicos. A menudo empezamos nuestros eventos con una charla de 10 minutos sobre «superpoderes
+porque creemos que la vida (tecnológica) es mucho más que programar.
```
Comment (cmaureir) [inline content/pages/es/meetups.rst]:
```suggestion
Tenemos diferentes niveles en los diferentes meetups, desde principiantes hasta expertos en temas específicos. A menudo comenzamos nuestros eventos con una charla de 10 minutos sobre un "superpoder sin programar" porque creemos que la vida (tecnológica) es mucho más que programar.
```

**Review by cmaureir — CHANGES_REQUESTED** [review]

**Review by cmaureir — APPROVED** [review]

**Comment by cmaureir** [comment]
¡Gracias!

**Pull request merged**